### PR TITLE
Working on Contact Form

### PR DESCRIPTION
**Pull request by mackenzieleach**
Netlify is now recognizing the contact form in our dashboard and I was able to successfully submit a message and see in on Netlify!

### CODE REVIEW

File: __tests__/home.test.js [hunk 1]
```diff
@@ -45,13 +45,11 @@ describe('Text Rendering Tests', () => {
...
   });
 
-  /* Since Linting, this is going to require a function to do the text matching */
-  /*     test('Website About Text Renders', () => {
-        render(<Home />)
-
-        expect(screen.getByText('Don\'t know what')).toBeInTheDocument();
-        cleanup()
-    }) */
+  test('Website About Text Renders', () => {
```
Comment (luuuk) [inline __tests__/home.test.js:48]: Im not sure what this is actually asserting on. Could you clarify this in the test header or on the expect statement?

**Review by luuuk — CHANGES_REQUESTED** [review]
Just need to add a bit more instruction on the test you added, then all good!

File: __tests__/home.test.js (hunk 1 above)
Comment (mackenzieleach) [reply __tests__/home.test.js:48]: Let me add a comment to the file & change the test title (it's in reference to the paragraph than renders on landing that explains the app). I'll ping when I commit!

Comment (luuuk) [reply __tests__/home.test.js]: Looks good!

**Review by luuuk — APPROVED** [review]

**Pull request merged**